x/txfees/keeper/keeper.go

```diff
@@ -9,27 +9,45 @@ import (
 	"github.com/cosmos/cosmos-sdk/store/prefix"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+
 	"github.com/osmosis-labs/osmosis/v7/x/txfees/types"
 )
 
-type (
-	Keeper struct {
-		cdc      codec.Codec
-		storeKey sdk.StoreKey
-
-		spotPriceCalculator types.SpotPriceCalculator
-	}
-)
+type Keeper struct {
+	cdc      codec.Codec
+	storeKey sdk.StoreKey
+
+	accountKeeper             types.AccountKeeper
+	bankKeeper                *bankkeeper.BaseKeeper
+	epochKeeper               types.EpochKeeper
+	gammKeeper                types.GammKeeper
+	spotPriceCalculator       types.SpotPriceCalculator
+	feeCollectorName          string
+	nonNativeFeeCollectorName string
+}
 
 func NewKeeper(
 	cdc codec.Codec,
+	accountKeeper types.AccountKeeper,
+	bankKeeper *bankkeeper.BaseKeeper,
```

Comment:
why are we relying on direct import of x/bank keeper? We should utilize the interface we defined in `expected_keepers.go`.

Reply:
Great catch - adding in next commit